server/src/main/java/io/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -293,12 +293,8 @@ private void drop(
 
     // Make sure we have enough loaded replicants in the correct tiers in the cluster before doing anything
```

Comment:
With method extracted, this comment line doesn't make much sense to me

Reply:
:+1: